src/containers/LocationPage/LocationPage.jsx

```diff
@@ -531,10 +532,13 @@ class LocationPage extends PureComponent {
   }
 
   renderBreadCrumbs() {
-    const { locationInfo } = this.props;
+    const { locationInfo, location } = this.props;
 
     return (
-      <Breadcrumbs info={locationInfo} />
+      <Breadcrumbs
+        info={locationInfo}
+        query={queryRebuild(location.query, urlQueryConfig)}
```

Comment:
Causes unnecessary re-renders of Breadcrumbs, but breadcrumbs is so simple it doesn't matter.